### PR TITLE
removing useage of photo-decomp

### PR DESCRIPTION
**Pull request by K20shores**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ TUV-x
 
 TUV-x: A photolysis rate constant calculator.
 
-[![License](https://img.shields.io/github/license/NCAR/photo-decomp.svg)](https://github.com/NCAR/photo-decomp/blob/main/LICENSE)
-[![CI Status](https://github.com/NCAR/photo-decomp/actions/workflows/test.yml/badge.svg)](https://github.com/NCAR/photo-decomp/actions/workflows/test.yml)
-[![codecov](https://codecov.io/gh/NCAR/photo-decomp/branch/main/graph/badge.svg?token=H46AAEAQF9)](https://codecov.io/gh/NCAR/photo-decomp)
+[![License](https://img.shields.io/github/license/NCAR/tuv-x.svg)](https://github.com/NCAR/tuv-x/blob/main/LICENSE)
+[![CI Status](https://github.com/NCAR/tuv-x/actions/workflows/test.yml/badge.svg)](https://github.com/NCAR/tuv-x/actions/workflows/test.yml)
+[![codecov](https://codecov.io/gh/NCAR/tuv-x/branch/main/graph/badge.svg?token=H46AAEAQF9)](https://codecov.io/gh/NCAR/tuv-x)
```
Comment (K20shores) [inline README.md:8]: Looks like the code-cov badge needs to have a specific URL and the one I changed it to doesn't work if you look at this branch's readme. Do you know about this, @mattldawson?

Comment (mattldawson) [reply README.md:8]: it looks like the token is the same on the codecov site for the new repo, and if I go to https://codecov.io/gh/NCAR/tuv-x/branch/main/graph/badge.svg?token=H46AAEAQF9 I can see the badge, and if I go to https://codecov.io/gh/NCAR/tuv-x I get redirected to https://app.codecov.io/gh/NCAR/tuv-x and can see the code coverage for NCAR/tuv-x. Maybe it just took a while to update?

**Review by mattldawson — APPROVED** [review]
looks good to me!

File: README.md (hunk 1 above)
Comment (mattldawson) [reply README.md:8]: and I just checked the README on this branch and it seems to work

**Pull request merged**